fluent-plugin-datadog.gemspec

```diff
@@ -21,7 +21,12 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 1.5"
```

Comment:
Why removing bundler from development dependency?
Most of Fluentd developers use bundler to manage plugin dependencies.
I would like to use bundler if available to manage dependencies when developing and testing.

Reply:
Mistake from my side, updated.